Description: Address the pointer mismatch in set_process
Author: Stephen Kitt <skitt@debian.org>

patch from: https://sources.debian.org/patches/schedtool/1.3.0-5/pointer-mismatch.patch/

--- a/schedtool.c
+++ b/schedtool.c
@@ -422,19 +422,19 @@
 	struct sched_param p;
 	int ret;
 
-	char *msg1="could not set PID %d to %s";
-	char *msg2="could not set PID %d to raw policy #%d";
-
 	p.sched_priority=prio;
 
 	/* anything other than 0 indicates error */
 	if((ret=sched_setscheduler(pid, policy, &p))) {
 
-                /* la la pointer mismatch .. lala */
-		decode_error((CHECK_RANGE_POLICY(policy) ? msg1 : msg2),
-			     pid,
-			     (CHECK_RANGE_POLICY(policy) ? TAB[policy] : policy)
-			    );
+		if (CHECK_RANGE_POLICY(policy)) {
+			decode_error("could not set PID %d to %s",
+				     pid, TAB[policy]);
+		} else {
+			decode_error("could not set PID %d to raw policy #%d",
+				     pid, policy);
+		}
+
 		return(ret);
 	}
 	return(0);
